test/v3/createPool/gyroECLP/gyroECLP.integration.test.ts

```diff
@@ -65,26 +66,26 @@ describe('GyroECLP - create & init', () => {
             .extend(walletActions);
         testAddress = (await client.getAddresses())[0];
 
-        await setTokenBalances(
-            client,
-            testAddress,
-            [DAI.address, WETH.address],
-            [DAI.slot!, WETH.slot!],
-            [parseUnits('10000', 18), parseUnits('10000', 18)],
-        );
+        // await setTokenBalances(
+        //     client,
+        //     testAddress,
+        //     [DAI.address, WETH.address],
+        //     [DAI.slot!, WETH.slot!],
+        //     [parseUnits('10000', 18), parseUnits('10000', 18)],
+        // );
 
-        await approveSpenderOnTokens(
-            client,
-            testAddress,
-            [DAI.address, WETH.address],
-            PERMIT2[chainId],
-        );
+        // await approveSpenderOnTokens(
+        //     client,
+        //     testAddress,
+        //     [DAI.address, WETH.address],
+        //     PERMIT2[chainId],
+        // );
         const eclpParams = {
-            alpha: parseUnits('2378', 18),
-            beta: parseUnits('2906', 18),
-            c: parseUnits('0.000378501108390785', 18),
-            s: parseUnits('0.999999928368452908', 18),
-            lambda: parseUnits('100000', 18),
+            alpha: parseUnits('450', 18),
+            beta: parseUnits('650', 18),
+            c: parseUnits('0.001785711438600104', 18),
+            s: parseUnits('0.999998405616057996', 18),
+            lambda: parseUnits('1', 18),
```

Comment:
using `lambda` of 1 the SDK test and tenderly simulation fails

https://www.tdly.co/shared/simulation/32b69896-f804-43e1-b32d-b9fc2eda558c

Reply:
using `lambda` of `2` the SDK create ECLP test passes and tenderly simulation success

https://dashboard.tenderly.co/balancer/sdk-team/simulator/b7817f65-f43b-44a5-a88e-a7ba70475015